### PR TITLE
Fixes #20988: Add a system-update technique for classic agents

### PR DESCRIPTION
**Pull request by Fdall**
https://issues.rudder.io/issues/20988

### CODE REVIEW

**Comment by Fdall** [comment]
Commit modified

**Comment by Fdall** [comment]
Commit modified

**Comment by Fdall** [comment]
Commit modified

**Comment by Fdall** [comment]
Commit modified

**Comment by Fdall** [comment]
PR updated with a new commit

File: techniques/applications/systemUpdate/1.0/metadata.xml [hunk 1]
```diff
@@ -0,0 +1,103 @@
+<TECHNIQUE name="SystemUpdate">
+  <DESCRIPTION>This technique can handle simple system wide updates on different Linux based systems.
+### Configuration
+
+Each update is unqiue and *must* be tagged by a unique `campaign name`. The name will be used to reference the scheduled system update by the agent.
```
Comment (peckpeck) [inline techniques/applications/systemUpdate/1.0/metadata.xml:5]: unique

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
PR updated with a new commit

**Review by peckpeck — APPROVED** [review]

**Pull request merged**